PyInstaller/depend/dylib.py

```diff
@@ -34,7 +37,15 @@
 # TODO convert _excludes and includes into type 'set()'.
 # Regex excludes
 # Ignoring some system libraries speeds up packaging process
-_excludes = {}
+_excludes = {
+    # Ignore annoying warnings like:
+    # 'W: library kernel32.dll required via ctypes not found'
+    # 'W: library coredll.dll required via ctypes not found'
+    r'^coredll\.dll$': 1,
+    r'^kernel32\.dll$': 1,
+    r'^kernel32$': 1,
```

Comment:
Theses are detected even on linux. I added '^kernel32$' here because I have a warning "W: library kernel32 required via ctypes not found". Not sure if it's good